### PR TITLE
feat(agent-tester): allow user to test the layout 

### PR DESCRIPTION
**Pull request by Scra3**

### CODE REVIEW

**Comment by Scra3** [comment]
https://app.clickup.com/t/86c0thy23

File: packages/agent-nodejs-testing/package.json [hunk 1]
```diff
@@ -28,18 +28,18 @@
...
+    "@forestadmin/forestadmin-client": "1.34.2",
+    "sequelize": "6.37.5",
+    "sqlite3": "^5.1.7"
   },
   "peerDependencies": {
-    "@forestadmin/agent": ">=1.41.1",
-    "@forestadmin/datasource-customizer": ">=1.47.0",
-    "@forestadmin/datasource-toolkit": ">=1.34.0",
-    "@forestadmin/forestadmin-client": ">=1.27.8"
+    "@forestadmin/agent": ">=1.52.0",
```
Comment (Scra3) [inline packages/agent-nodejs-testing/package.json:40]: Not really sure of the version where we unlock the form layout feature, that why I have put the last one.

File: packages/agent-nodejs-testing/src/integrations/action-layout/testable-action-layout-element.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { NotFoundElementError, NotRightElementError } from './errors';
+import TestableActionLayoutInput from './testable-action-layout-input';
+
+export default class TestableActionLayoutElement extends TestableActionLayoutInput {
```
Comment (EnkiP) [inline packages/agent-nodejs-testing/src/integrations/action-layout/testable-action-layout-element.ts:4]:
```suggestion
export default class TestableActionLayoutElement extends TestableActionLayoutBase {
```

File: packages/agent-nodejs-testing/src/integrations/action-layout/testable-action-layout-input.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { ForestServerActionFormLayoutElement } from '@forestadmin/forestadmin-client';
+
+import { NotRightElementError } from './errors';
+
+export default class TestableActionLayoutInput {
```
Comment (EnkiP) [inline packages/agent-nodejs-testing/src/integrations/action-layout/testable-action-layout-input.ts:5]:
```suggestion
export default class TestableActionLayoutInput extends TestableActionLayoutElement {
```

File: packages/agent-nodejs-testing/src/integrations/action-layout/testable-action-layout-page.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { ForestServerActionFormLayoutElement } from '@forestadmin/forestadmin-client';
+
+import TestableActionLayoutBase from './testable-action-layout-base';
+
+export default class TestableActionLayoutPage extends TestableActionLayoutBase {
```
Comment (EnkiP) [inline packages/agent-nodejs-testing/src/integrations/action-layout/testable-action-layout-page.ts:5]:
```suggestion
export default class TestableActionLayoutPage extends TestableActionLayoutElement {
```

File: packages/agent-nodejs-testing/src/integrations/action-layout/testable-action-layout-element.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  isRow() {
+    return this.layoutItem?.component === 'row';
+  }
+
+  isHTMLBlock() {
+    return this.layoutItem?.component === 'htmlBlock';
+  }
+
```
Comment (EnkiP) [inline packages/agent-nodejs-testing/src/integrations/action-layout/testable-action-layout-element.ts:16]: add isInput()

**Review by EnkiP — APPROVED** [review]

**Pull request merged**